src/python/pants/core/goals/export.py

```diff
@@ -163,7 +163,9 @@ async def export(
                 env={"PATH": environment.get("PATH", ""), **cmd.extra_env},
                 run_in_workspace=True,
             )
-            ipr = await Effect(InteractiveProcessResult, InteractiveProcess, ip)
+            ipr = await Effect(  # noqa: PNT30: requires triage
```

Comment:
`Effect`s will tend to be inherently sequential: ditto the case in `src/python/pants/core/goals/deploy.py` although it is less obvious there due to indirection. If skipping `await Effect` entirely is possible, that would be handy.